source/extensions/filters/network/zookeeper_proxy/filter.cc

```diff
@@ -18,8 +18,18 @@ namespace ZooKeeperProxy {
 
 ZooKeeperFilterConfig::ZooKeeperFilterConfig(const std::string& stat_prefix,
                                              const uint32_t max_packet_bytes, Stats::Scope& scope)
-    : scope_(scope), max_packet_bytes_(max_packet_bytes), stat_prefix_(stat_prefix),
-      stats_(generateStats(stat_prefix, scope)) {}
+    : scope_(scope), max_packet_bytes_(max_packet_bytes), stats_(generateStats(stat_prefix, scope)),
+      stat_name_set_(scope.symbolTable()), stat_prefix_(stat_name_set_.add(stat_prefix)),
+      auth_(stat_name_set_.add("auth")) {
+  // https://zookeeper.apache.org/doc/r3.1.2/zookeeperProgrammers.html#sc_BuiltinACLSchemes
+  // lists commons schemes: "world", "auth", "digest", "host", and "ip". These are used in
```

Comment:
@rgs1 does this match your understanding of common stat names?

Reply:
Mostly correct, but lets reference a more updated link instead:

https://zookeeper.apache.org/doc/r3.5.4-beta/zookeeperProgrammers.html#sc_BuiltinACLSchemes

And include the `x509` scheme as well. 

[or, i can do it in a follow-up]

Reply:
good catch, thanks.